Project/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAgent.cs

```diff
@@ -37,6 +57,14 @@ public class WalkerAgent : Agent
 
     public override void Initialize()
     {
+        //Spawn an orientation cube
+        Vector3 oCubePos = hips.position;
+        oCubePos.y = -.45f;
+        m_OrientationCube = Instantiate(Resources.Load<GameObject>("OrientationCube"), oCubePos, Quaternion.identity);
```

Comment:
Isn't it simpler to always have the orientation cube attached since it will always be attached in Initialize anyway ?
This code could break if the "OrientationCube" is renamed (which is not ideal)

Reply:
removed. refactored to include the cube as part of the prefab